new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleDomainValidation.java

```diff
@@ -30,8 +32,19 @@ public void testCreatingDomain() {
         String domainYAML= "something";
 
         // get a new unique namespace
-        String namespace = createUniqueNamespace();
-        logger.info(String.format("Got a new namespace called %s", namespace));
+        String namespace = null;
+        try {
+            namespace = createUniqueNamespace();
+            logger.info(String.format("Got a new namespace called %s", namespace));
+        } catch (ApiException e) {
+            // TODO: test in the calling method where we say something like
+            //
+            //  assert.DoesNotThrow(whatever(), ApiException.class, e,
+            //    String.format("could not do whatever, got exception %s", e))
+            //
+            //  so we have the exception and we can print out a meaningful error message
+        }
+
```

Comment:
For the moment, updated the test to surround with try-catch until we learn how to handle exceptions with JUnit 5 assertions, as noted from Mark.